11ty/definitions/transfeminine.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Transfeminine
+slug: transfeminine
+speech: adj
+defined: true
+---
```

Comment:
```suggestion
reading:
 - text: What does transmasculine and transfeminine mean
   href: http://allaboutgender.com/2018/07/08/what-does-transmasculine-and-transfeminine-mean/
---
```

Reply:
🤔 That link looks okay at first pass, although it does use "transman" and "transwoman" (as opposed to "trans man" and "trans woman") which is a minor red flag. I'll see if I can find something that's written by a transfeminine (or transmasculine) person themselves, but if I can't this is good.

Reply:
Ah, I did not know that as a red flag, and now I do, thank you for educating me! I'm going to merge this in as I think it is okay for it to have no further readings for now (and would much prefer it have no harmful further readings!).

If you have time/energy/desire to find a better resource, I invite you to open a new PR! 💙